### DISCUSSION TITLE
Generating a report from the command line.

### DISCUSSION THREAD

**Discussion by Turystaa666** [FortuneN/FineCodeCoverage#460] [Ideas]
Hi,
I need to merge several branches before building the solution, running the tests and getting the report.
I would like the entire process to be run from the command line. 
Please add the ability to of generating a report from the command line.

**Comment by tonyhallett** [comment]
This is not going to happen anytime soon. 
FCC works in conjunction with visual studio.  In particular the options and interception of runsettings.
There are methods of using the command line - see https://reportgenerator.io/getstarted
